app/controllers/admin/extensions_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Admin
+  class ExtensionsController < Admin::ApplicationController
+    layout "admin"
+
+    EXTENSIONS = [
+      Extension.new(
+        name: "Listings",
+        description: "Once turned on, Listings can be accessed via /listings and members " \
+                     "will be able to add listings. " \
+                     "Learn more about Listings <a href='https://admin.forem.com/docs/advanced-customization/listings'>here</a>.",
```

Comment:
```suggestion
                     "<a href='https://admin.forem.com/docs/advanced-customization/listings'>Learn more about Listings</a>.",
```

(suggestion / not blocking) - links with names like "here" are very difficult for screen reader users to understand - maybe we can just make the whole thing a link instead?